commerce-cloud-b2c-data/1.0/modules/ROOT/pages/commerce-cloud-b2c-data-connector-examples.adoc

```diff
@@ -0,0 +1,181 @@
+= Commerce Cloud B2C Data Connector Example - Mule 4
+
+
+This use case shows how a guest user logs in to the Shop API and submits a basket of products, with shopping details. The successful submission of this order leads to registering the guest user as a customer. 
```

Comment:
@valkyrie69 Can u pls refer the source of this doc? I looked this doc from apisero and could not find any reference to 'Shop'... https://github.com/Apisero-Connectors/mule4-salesforce-commerce-cloud-data-connector-documentation/blob/master/modules/ROOT/pages/data-api/data-api-connector-user-guide/data-api-connector-examples.adoc